### DISCUSSION TITLE
problem with registration on students gihub page

### DISCUSSION THREAD

**Discussion by konstantinosDMS** [community/community#33516] [GitHub Education]
after completing, veryfying my email address on developer pack , the page ask me for id card veyfication, my card looks like in front has my photo and several elements like university's name, back has the issuing date and other stuff. When i upload the first side of my id card the web site ask my for the issuing date, when i upload seconf side the side ask me for my photo. How can i resolve the problem since i am able to upload only one side of my id card?

**Comment by FahdSeddik** [comment]
Hi @konstantinosDMS 
Try putting both in one image. Also check this [link](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved). I hope this helps!

**Reply by FahdSeddik** [reply]
Make sure to mark it as an answer if it does so it reaches other people easier.

**Reply by SevenSilver** [reply]
Attach date side image.

**Comment by cosmicqbit** [comment]
Scan the backside or click a mirror selfie of the backside of your ID card and the print the same. When the site asks for say Issue date you can hold the printed copy in one hand with and extra heading added at the top reading **Backside** and the original ID card in another hand with your pic as a front side.
